packages/eslint-config-react-app/jest.js

```diff
@@ -12,21 +12,62 @@
 // and we try not to use "ERROR" level at all.
 
 module.exports = {
-  plugins: ['jest'],
+  plugins: ['jest', 'jest-dom', 'testing-library'],
   overrides: [
     {
       files: ['**/__tests__/**/*', '**/*.{spec,test}.*'],
       env: {
         'jest/globals': true,
       },
       // A subset of the recommended rules:
-      // https://github.com/jest-community/eslint-plugin-jest#rules
       rules: {
-        'jest/expect-expect': 'warn',
-        'jest/no-identical-title': 'warn',
-        'jest/valid-describe': 'warn',
-        'jest/valid-expect': 'warn',
-        'jest/valid-expect-in-promise': 'warn',
+        // https://github.com/jest-community/eslint-plugin-jest
+        'jest/expect-expect': 'off',
+        'jest/no-commented-out-tests': 'warn',
+        'jest/no-conditional-expect': 'error',
+        'jest/no-deprecated-functions': 'warn',
+        'jest/no-disabled-tests': 'warn',
+        'jest/no-done-callback': 'error',
+        'jest/no-export': 'error',
+        'jest/no-focused-tests': 'warn',
+        'jest/no-identical-title': 'error',
+        'jest/no-interpolation-in-snapshots': 'error',
+        'jest/no-jasmine-globals': 'error',
+        'jest/no-jest-import': 'error',
+        'jest/no-mocks-import': 'error',
+        'jest/no-standalone-expect': 'off',
+        'jest/no-test-callback': 'off',
+        'jest/no-test-prefixes': 'warn',
+        'jest/valid-describe': 'error',
+        'jest/valid-expect': 'error',
+        'jest/valid-expect-in-promise': 'error',
+        'jest/valid-title': 'warn',
+
+        // https://github.com/testing-library/eslint-plugin-jest-dom
+        'jest-dom/prefer-checked': 'warn',
+        'jest-dom/prefer-empty': 'warn',
+        'jest-dom/prefer-enabled-disabled': 'warn',
+        'jest-dom/prefer-focus': 'warn',
+        'jest-dom/prefer-required': 'warn',
+        'jest-dom/prefer-to-have-attribute': 'warn',
+        'jest-dom/prefer-to-have-style': 'warn',
+        'jest-dom/prefer-to-have-text-content': 'warn',
```

Comment:
@ianschmitz and @iansu all rules for this plugin are `prefer`. Perhaps we should just remove it?

Reply:
All rules are recommended in the `jest-dom` plugin.

If not adhered there won't be any error, but I still think it's a good idea to at least warn the users and "teach" them how they should use `jest-dom`.

Reply:
I understand sorry, it was more about aligning with the philosophy of the current ESLint config - which has always tried to avoid preferential rules.
https://github.com/facebook/create-react-app/blob/master/packages/eslint-config-react-app/index.js

But again, let's see what the rest of the team think :) Let's not do anything until we get that feedback.

@Timer may also have some thoughts on this.

Reply:
We'll remove this for now, and consider adding in later - perhaps as an optional/extended set of rules.